plugins/modules/win_acl.ps1

```diff
@@ -89,6 +89,7 @@ $state = Get-AnsibleParam -obj $params -name "state" -type "str" -default "prese
 
 $inherit = Get-AnsibleParam -obj $params -name "inherit" -type "str"
 $propagation = Get-AnsibleParam -obj $params -name "propagation" -type "str" -default "None" -validateset "InheritOnly", "None", "NoPropagateInherit"
+$follow_links = $True
```

Comment:
This should be a parameter, we typically use `follow` for this type of option as per https://docs.ansible.com/ansible/latest/collections/ansible/builtin/file_module.html#parameter-follow. Don't forget to update the `win_acl.py` docs for this new parameter.